package com.pm;

public class Ex04 {
	public static void func01() throws ArithmeticException{
		int su = 0/0;
		
	}
	public static void main(String[] args) {
		try{
			func01();
		}catch (ArithmeticException e) {
			System.out.println("예외 발생");
		}
			
	}
}
